src/sagemaker/fw_utils.py

```diff
@@ -28,26 +28,28 @@
 """
 
 
-def create_image_uri(region, framework, instance_type, py_version='py2', tag='1.0', account='520713654638'):
+def create_image_uri(region, framework, instance_type, framework_version, py_version='py2', account='520713654638'):
```

Comment:
I've notice that `create_image_uri` does not have unit tests. Can you write unit tests for this function?

Reply:
it's doesn't, it should 👍